src/pages/Profile/index.tsx

```diff
@@ -0,0 +1,18 @@
+import { useAppDispatch } from 'app/hooks';
+import { logout } from 'features/User/userSlice';
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+
+export function Profile() {
+  const dispatch = useAppDispatch();
+  const navigate = useNavigate();
+  const logoutHandler = () => {
+    dispatch(logout());
+    navigate('/');
+  };
+  return (
+    <div>
+      <button onClick={logoutHandler}>Logout</button>
```

Comment:
suggestion: avoid this component just for logout if there is no further updates on this